roadmap.md

```diff
@@ -0,0 +1,35 @@
+Roadmap Zombienet v2
+
+## Infra
+- Chaos testing, add examples and explore possibilities in `native` and `podman` provider
```

Comment:
This is top priority for parachains. We want to roll out a separate CI pipeline to do these long duration tests.